cmd/merge/transactions.go

```diff
@@ -183,3 +211,58 @@ func mergeTransactions(cCtx *cli.Context) error {
 	log.Infof("Finished processing CSV files, wrote %s transactions", printer.Sprintf("%d", cntTxWritten))
 	return nil
 }
+
+func updateInclusionStatus(log *zap.SugaredLogger, checkNodeURIs []string, txs map[string]*common.TxSummaryEntry) (err error) {
+	// Load inclusion status for all transactions
+	workers := numRPCWorkers
+	txC := make(chan *common.TxSummaryEntry, 2000000)
```

Comment:
This code below will block if there are more than 2kk of txs.

Is this intentional?